src/parser/jobs/ast/modules/CelestialIntersection.tsx

```diff
@@ -43,7 +48,7 @@ export default class CelestialIntersection extends Module {
 	onComplete() {
 		const holdDuration = this.uses === 0 ? this.parser.currentDuration : this.totalHeld
 		const usesMissed = Math.floor(holdDuration / (this.data.actions.CELESTIAL_INTERSECTION.cooldown))
-		const fightDuration = this.parser.fight.end_time - this.parser.fight.start_time
+		const fightDuration = this.parser.pull.duration
 		const maxUses = (fightDuration / this.data.actions.CELESTIAL_INTERSECTION.cooldown) - 1
```

Comment:
```suggestion
		const maxUses = (this.parser.pull.duration / this.data.actions.CELESTIAL_INTERSECTION.cooldown) - 1
```

Condense variable since there's no calculation going on now.